indico/modules/events/management/controllers/protection.py

```diff
@@ -114,13 +114,11 @@ def _process(self):
 class RHEventPrincipals(PrincipalsMixin, RHManageEventBase):
     ALLOW_LOCKED = True
 
-    def _process_args(self):
-        RHManageEventBase._process_args(self)
-        args = parser.parse({
-            'values': PrincipalDict(allow_groups=True, allow_external_users=True, allow_event_roles=True,
-                                    allow_category_roles=True, event_id=self.event.id, missing={})
-        })
-        self.values = args['values']
+    @use_rh_kwargs({'values': PrincipalDict(allow_groups=True, allow_external_users=True, allow_event_roles=True,
```

Comment:
I'd wrap after `{` since we usually do this in other places where we have `use_*()`